PayPalNative/src/main/java/com/braintreepayments/api/PayPalNativeVaultRequest.java

```diff
@@ -0,0 +1,6 @@
+package com.braintreepayments.api;
+
+//TODO: once that native checkout supports vault, this class should be eliminated,
```

Comment:
I think we still need this class even with the vault fallback. We can probably remove this TODO and add doc strings from `PayPalVaultRequest` though.